website/scripts/core/blueprintService.js

```
@@ -1,695 +1,207 @@
-    updateCursor(x, y) {
-        const user = this.users.get(this.localUserId);
-        if (user) {
-            this.users.set(this.localUserId, {
-                ...user,
-                cursor: { x, y },
-                lastSeen: Date.now()
-            });
+        switch (type) {
+            case 'node_update':
+                const existingNode = this.nodes.get(data.id);
+                if (!existingNode || data.lastModified > existingNode.lastModified) {
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)